program-book-club.html

```diff
@@ -9,14 +9,27 @@ <h1>Book Club Programme</h1>
 <div class="container">
     <div class="content-constrained">
         <div class="section">
-            <p>What sets our book club apart are the lively conversations, valuable insights and an opportunity to connect with like-minded individuals. The books we read are selected by the community, offering a diverse selection of tech books and essential soft skills topics.</p>
-            <p>The book club meets monthly to share views, exchange ideas, and inspire personal and professional growth.</p> 
-            <p>Join our <a href="https://womencodingcommunity.slack.com/archives/C06VB4MH17G" target="_blank">slack channel #prog_book_club</a> to be part of this vibrant community.</p>
+            <p>What sets our book club apart are the lively conversations, valuable insights and an opportunity to
+                connect with like-minded individuals. The books we read are selected by the community, offering a
+                diverse selection of tech books and essential soft skills topics.</p>
+            <p>The book club meets monthly to share views, exchange ideas, and inspire personal and professional
+                growth.</p>
+            <p>Join our <a href="https://womencodingcommunity.slack.com/archives/C06VB4MH17G" target="_blank">slack
+                channel #prog_book_club</a> to be part of this vibrant community.</p>
         </div>
+
         <section class="section" id="what_we_are_reading">
             <h2>What we are reading</h2>
-            <p><a href="https://www.goodreads.com/book/show/4099.The_Pragmatic_Programmer" target="_blank">The Pragmatic Programmer: From Journeyman to Master</a>, Andrew Hunt & David Thomas</p>
-            <p>Essential guide for creating working and maintainable code amidst the complexities of modern software development.</p>
+            <ul>
+                {% for book in site.data.book_club %}
+                {% if book.status == 'reading' %}
```

Comment:
Good idea with the book status🚀 We can update read books too as we go along, once the book club meetings have happened.

Lily and I discussed using a card component like the one we created for the blog to list the books. A layout of 4 cards per row to make it more compact.

Reply:
I like that!! we could also add the book photo would be really nice. We can keep this issue open. Or close and create another one just for the style.